pages/index.js

```diff
@@ -149,13 +149,14 @@ const EditorContainer = styled.div`
 `
 
 const Editor = styled(LiveEditor)`
-  ${editorMixin}
-  height: 24rem;
+  ${editorMixin} height: 24rem;
```

Comment:
one more missing `;` here 🙂  and there are a few more later on.

It might be worth fixing all this in the source and _then_ run prettier in just one commit on top of all other changes.